src/directwrite/mod.rs

```diff
@@ -72,50 +75,42 @@ impl DirectWriteRasterizer {
             dwrote::DWRITE_MEASURING_MODE_NATURAL,
             0.0,
             0.0,
-        )
-        .map_err(Error::DirectWriteError)?;
+        )?;
 
-        let bounds = glyph_analysis
-            .get_alpha_texture_bounds(dwrote::DWRITE_TEXTURE_CLEARTYPE_3x1)
-            .map_err(Error::DirectWriteError)?;
+        let bounds =
+            glyph_analysis.get_alpha_texture_bounds(dwrote::DWRITE_TEXTURE_CLEARTYPE_3x1)?;
 
-        let buf = glyph_analysis
-            .create_alpha_texture(dwrote::DWRITE_TEXTURE_CLEARTYPE_3x1, bounds)
-            .map_err(Error::DirectWriteError)?;
+        let buffer = BitmapBuffer::RGB(
+            glyph_analysis.create_alpha_texture(dwrote::DWRITE_TEXTURE_CLEARTYPE_3x1, bounds)?,
+        );
 
         Ok(RasterizedGlyph {
-            c,
+            character,
             width: (bounds.right - bounds.left) as i32,
             height: (bounds.bottom - bounds.top) as i32,
             top: -bounds.top,
             left: bounds.left,
-            buf: BitmapBuffer::RGB(buf),
+            buffer,
         })
     }
 
     fn get_loaded_font(&self, font_key: FontKey) -> Result<&Font, Error> {
-        self.fonts.get(&font_key).ok_or(Error::FontNotLoaded)
+        self.fonts.get(&font_key).ok_or(Error::UnknownFontKey)
     }
 
-    fn get_glyph_index(&self, face: &FontFace, c: char) -> Result<u16, Error> {
-        let idx = *face
-            .get_glyph_indices(&[c as u32])
-            .first()
-            // DirectWrite returns 0 if the glyph does not exist in the font.
-            .filter(|glyph_index| **glyph_index != 0)
-            .ok_or_else(|| Error::MissingGlyph(c))?;
-
-        Ok(idx)
+    fn get_glyph_index(&self, face: &FontFace, character: char) -> u16 {
+        // DirectWrite returns 0 if the glyph does not exist in the font.
```

Comment:
This comment doesn't make a lot of sense anymore. If we would `unwrap` here, sure. But we make sure that everything works out and do `unwrap_or(0)` (which is good).

So if we talk about anything, we should probably talk about why we return `0` in the unwrap case? But I think we can just remove this comment completely.

Previously it made sense because we explicitly *filtered* the `0` out, but now we don't do that anymore and the constant makes it clear what the unwrap does. So there's no need to go into any more detail.